### ISSUE TITLE
Ui class not useful after adding GUI ?

### ISSUE THREAD

**Issue by Jacky142857** [nus-cs2103-AY2122S2/forum#124]
I actually don't find Ui class that useful to be extracted out after adding GUI. We don't need a scanner anymore. If I want to print some general message, I can just print that message in where it should be printed, instead of sending the message from an Ui object. I looked at other people's code and it seems that lots of their ui class is not useful either.

**Comment by domlimm** [comment]
I left my `Ui` class there :') And yes, it's not being used at all 🤣 I think you could scrap the entire `Ui` class if you'd like! I remember it was stated that we will not be using it anymore when the `GUI` was first introduced.

**Comment by ryantanlien** [comment]
I left my `Ui` class there as well. I turned it into a handler/manager sort of class that acts as a middle layer between the Butler Chat Bot and the GUI, so there is a separation of concerns. I have placed the handleUserInput that was previously inside the Butler class in the JavaFX tutorial inside the singleton Ui class. That way we can change the GUI based on Butler's outputs instead of hard-coding everything which would make refactoring difficult :)

After all the GUI class's job is just to present the GUI right? 
Just my two cents haha.
